common/gnome-shell/3.30/sass/_common.scss

```diff
@@ -1009,7 +1009,7 @@ StScrollBar {
   .screencast-indicator { color: $error_color; }
 
   .clock-display > * > *:last-child {
-    color: $selected_bg_color;
+    color: $selected_fg_color;
```

Comment:
Why not just removing this line and leaving it to the default color? I guess there should be no reason to assign to it a "selected color"...

Reply:
Yeah, I didn't think about that. If there is no difference anyway, it would be better to remove it.

Reply:
ta. removed and merged